mlflow/metrics/genai/prompts/v1.py

```diff
@@ -76,3 +76,29 @@ def to_dict(self):
 
     def _format_examples(self):
         return "\n".join(map(str, self.examples))
+
+
+@dataclass
+class CorrectnessMetric:
+    definition = (
+        "Correctness is assessed by whether answers align exactly with established truth for "
+        "questions requiring precise responses, and by the similarity in meaning and description "
+        "to the established truth for questions allowing varied descriptions."
+    )
+
+    # TODO: modify this for new definition of correctness
+    grading_prompt = (
+        "Correctness: If the answer correctly answer the question, below "
+        "are the details for different scores: "
+        "- Score 1: the answer is completely incorrect, doesn’t mention anything about "
+        "the question or is completely contrary to the correct answer. "
+        "- Score 2: the answer provides some relevance to the question and answer "
+        "one aspect of the question correctly. "
+        "- Score 3: the answer mostly answer the question but is missing or hallucinating "
+        "on one critical aspect. "
+        "- Score 5: the answer correctly answer the question and not missing any "
+        "major aspect"
+    )
+
+    variables = ["ground_truth"]
+    parameters = {"temperature": 0.0}
```

Comment:
I think we can use the [default_parameters](https://github.com/mlflow/mlflow/pull/9673/files#diff-6f989efdf75d569970663449be1700307f6889952028f66ad632cfd7e47bc241L13) right now. No need to override for the metrics here.